pkgs/development/python-modules/pure-cdb/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, isPy3k, flake8 }:
+
+buildPythonPackage rec {
+  pname = "pure-cdb";
+  version = "3.1.1";
+
+  # Archive on pypi has no tests.
+  src = builtins.fetchGit {
+    url = "https://github.com/bbayles/python-pure-cdb";
+    rev = "a1f496a95b892c1304e637fb352f663c65eb2655";
+  };
+
+  checkInputs = [ flake8 ];
+  disabled = !isPy3k;
+
```

Please consider to add `pythonImportsCheck` as an additional safeguard. Even if you are running the tests.

```suggestion
  pythonImportsCheck = [ "cdblib" ];
```